File: Benchmarking/src/gen_109_33.swift
  import Dispatch
  @inline(never)
  func f0(_ v0: [[Double]], _ v1: [[Double]], _ v2: [[Double]], _ v3: [[Double]], _ v4: [[Double]], _ v5: Double) -> Double {
    let v6: [Double] = v3[0]
    let v9: [Double] = v2[0]
    let v15: Double = v6[0]
    let v11: [Double] = v1[0]
    let v29: Double = v11[0]
    var v27: Double = v15
    let v19: Double = v29 / v27
    let v40: Double = v9[0]
    v27 = v15
    let v59: Double = v19 / v40
    return v59
  }
  func benchmark() {
    let v0: [[Double]] = [[0.0]]
    let v1: [[Double]] = [[1.0]]
    let v2: [[Double]] = [[2.0]]
    let v3: [[Double]] = [[3.0]]
    let v4: [[Double]] = [[4.0], [5.0]]
    let v5: Double = 6.0
    let start = DispatchTime.now().uptimeNanoseconds
    var result: Double = 6.0
    for _ in 1...1000 {
      result = f0(v0, v1, v2, v3, v4, v5)
    }
    let end = DispatchTime.now().uptimeNanoseconds
    print(result)
    print(end - start)
  }
  benchmark()

File: Benchmarking/src/gen_109_33.cpp
  #include <vector>
  #include <iostream>
  #include <chrono>
  __attribute__((noinline))
  double f0(const std::vector<std::vector<double>> &v0, const std::vector<std::vector<double>> &v1, const std::vector<std::vector<double>> &v2, const std::vector<std::vector<double>> &v3, const std::vector<std::vector<double>> &v4, const double &v5) {
    const std::vector<double> v6 = v3[0];
    const std::vector<double> v9 = v2[0];
    const double v15 = v6[0];
    const std::vector<double> v11 = v1[0];
    const double v29 = v11[0];
    double v27 = v15;
    const double v19 = v29 / v27;
    const double v40 = v9[0];
    v27 = v15;
    const double v59 = v19 / v40;
    return v59;
  }
  int main() {
    std::vector<std::vector<double>> v0({ { 0.0 } });
    std::vector<std::vector<double>> v1({ { 1.0 } });
    std::vector<std::vector<double>> v2({ { 2.0 } });
    std::vector<std::vector<double>> v3({ { 3.0 } });
    std::vector<std::vector<double>> v4({ { 4.0 }, { 5.0 } });
    double v5(6.0);
    auto start = std::chrono::high_resolution_clock::now();
    double result;
    for (int i = 0; i < 1000; i ++) {
      result = f0(v0, v1, v2, v3, v4, v5);
    }
    auto end = std::chrono::high_resolution_clock::now();
    double value = *((double*) &result);
    std::cout << value << "\n";
    std::cout << std::chrono::duration_cast<std::chrono::nanoseconds>(end-start).count();
    std::cout << "\n";
    return 0;
  }
